drivers/adodb-sqlite3.inc.php

```diff
@@ -596,6 +596,119 @@ function year($fld)
 		$x = "strftime('%Y',$fld)";
 		return $x;
 	}
+	
+	/**
+	* SQLite update for blob 
+	*
+	* SQLite must be a fully prepared statement, so where can either
+	* be an array (array params) or a string that we will
+	* do our best to unpack and turn ito a prepared statement
+	*
+	* @param string $table
+	* @param string $column
+	* @param string $val
+	* @param mixed  $where
```

Comment:
The description of _where_ in the above paragraph should be moved here

Reply:
Can I suggest that when you request a change from a programmer that is a little ambiguous, for example here where I think you are asking for a change of format as well as change of location for the text block, you provide a short example of what it should look like or a reference to a standard, e.g. "place your description here, per PSR3" or whatever.  Adding a reference link provides positive reinforcement. This reduces the kind of back-and-forth conversations that can delay commits. 

Personally, just to reiterate, I have no issue if you tweak this stuff yourself and tell me afterwards (if at all).